torch/_inductor/codegen/common.py

```diff
@@ -1561,7 +1561,7 @@ def namedvar(
         bounds: ValueRanges[Any] = ValueRanges.unknown(),
         dtype: Optional[torch.dtype] = None,
     ) -> CSEVariable:
-        assert name not in self.varname_map, "duplicate name"
+        torch._check(name not in self.varname_map, lambda: f"duplicate name: {name}")
```

Comment:
```suggestion        
        torch._check_value(name not in self.varname_map, lambda: f"duplicate name: {name}")
```
So it raises a ValueError